src/AppInstallerRepositoryCore/Rest/Schema/1_0/Interface.cpp

```diff
@@ -144,8 +178,25 @@ namespace AppInstaller::Repository::Rest::Schema::V1_0
             std::vector<VersionInfo> versions;
             for (auto& manifestVersion : manifests)
             {
+                std::vector<std::string> packageFamilyNames;
+                std::vector<std::string> productCodes;
+
+                for (auto& installer : manifestVersion.Installers)
+                {
+                    if (!installer.PackageFamilyName.empty())
+                    {
+                        packageFamilyNames.emplace_back(installer.PackageFamilyName);
+                    }
+
+                    if (!installer.ProductCode.empty())
+                    {
+                        productCodes.emplace_back(installer.ProductCode);
+                    }

```

Comment:
Do you need to check duplicates before add? Or you have dedupe logic later?

Reply:
Or maybe we don't care..

Reply:
In GetMultiProperty() implementation in RestSource.cpp, there's no dedupe logic, I think it's better to dedupe here

Reply:
Added dedupe logic in the two places where pfn and product codes were added.